blivet/partitioning.py

```diff
@@ -409,7 +409,12 @@ def add_partition(disklabel, free, part_type, size, start=None, end=None):
         else:
             _size = size
 
-        alignment = disklabel.get_alignment(size=_size)
+        try:
+            alignment = disklabel.get_alignment(size=_size)
+        except AlignmentError:
+            alignment = disklabel.get_minimal_alignment()
+            _size = Size(alignment.grainSize)
```

Comment:
`_size` is used only in the `get_alignment` call above, we shouldn't need it here.

It might be useful to add a `log.debug` here just to be sure we know about the change to the minimal alignment.

Reply:
We do log which alignment we've chosen just below. Is that sufficient?